frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { DetailsBody, DetailItem } from '../../dashboard/details-card';
+import { withDashboardResources, DashboardItemProps } from '../with-dashboard-resources';
+import { InfrastructureModel, ClusterVersionModel } from '../../../models';
+import { referenceForModel } from '../../../module/k8s';
+import { FLAGS } from '../../../const';
+import { featureReducerName, flagPending } from '../../../reducers/features';
+import { RootState } from '../../../redux';
+import { getK8sGitVersion, getOpenShiftVersion } from '../../about-modal';
+import { FirehoseResource } from '../../utils';
+
+const getClusterName = (): string => {
```

Comment:
We should make sure we have a consistent implementation with #1721. I'm in favor of using the API server host as-is for the moment since we don't have access to a real cluster name.

@christianvogt fyi

Reply:
Im good with using API server as-is. Extracted the `getClusterName` function to `cluster-settings.ts` so it can be reused by https://github.com/openshift/console/pull/1721